pkg/pdutil/pd.go

```diff
@@ -441,6 +443,12 @@ func restoreSchedulers(ctx context.Context, pd *PdController, clusterCfg cluster
 	if err := pd.UpdatePDScheduleConfig(ctx, scheduleLimitCfg); err != nil {
 		return errors.Annotate(err, "fail to update PD schedule config")
 	}
+	if locationPlacement, ok := clusterCfg.scheduleCfg["enable-location-replacement"]; ok {
```

Comment:
How about adding `"enable-location-replacement"` to `pdScheduleLimitCfg`?

Reply:
The map `pdScheduleLimitCfg` is also used in `removeSchedulers`, and they also have some generic traits(e.g. both of their key are number), so I'm fearing adding `enable-location-replacement` would make extra overhead.

See here:

https://github.com/pingcap/br/blob/6734867f0fd1d6f12dc38c51fdd3ae11749bed01/pkg/pdutil/pd.go#L511-L522